src/KeyboardLayouts/README.adoc

```diff
@@ -0,0 +1,67 @@
+= Contributors =
+
+Original code (pre-library): https://github.com/NicoHood/HID[NicoHood/HID]
+
+* Links/Credits
+
+** http://www.fourwalledcubicle.com/LUFA.php[LUFA from Dean Camera]
+
+** https://github.com/harlequin-tech/arduino-usb[Darran's HID Projects]
+
+** http://www.imaginaryindustries.com/blog/?p=80[Connor's Joystick for the Leonardo]
+
+** http://stefanjones.ca/blog/arduino-leonardo-remote-multimedia-keys/[Stefan Jones Multimedia Keys Example]
+
+** https://github.com/aduitsis/ardumultimedia[Athanasios Douitsis Multimedia Keys Example]
+
+** https://github.com/arduino/Arduino/tree/master/hardware/arduino/firmwares/atmegaxxu2/arduino-usbserial[The Original Arduino Sources]
+
+** http://www.usblyzer.com/[USBlyzer]
+
+** https://www.mattairtech.com/index.php/development-boards/mt-db-u1.html[Mattairtechs 16u2 Lufa USB Core]
```

Comment:
```suggestion
** https://www.mattairtech.com/index.php/old-products/mt-db-u1.html[Mattairtechs 16u2 Lufa USB Core]
```
The URL changed since the time the source text was written.